yt_dlp/extractor/abematv.py

```diff
@@ -145,76 +146,14 @@ def abematv_license_open(self, url):
 
 
 class AbemaTVBaseIE(InfoExtractor):
-    def _extract_breadcrumb_list(self, webpage, video_id):
-        for jld in re.finditer(
-                r'(?is)</span></li></ul><script[^>]+type=(["\']?)application/ld\+json\1[^>]*>(?P<json_ld>.+?)</script>',
-                webpage):
-            jsonld = self._parse_json(jld.group('json_ld'), video_id, fatal=False)
-            if jsonld:
-                if jsonld.get('@type') != 'BreadcrumbList':
-                    continue
-                trav = traverse_obj(jsonld, ('itemListElement', ..., 'name'))
-                if trav:
-                    return trav
-        return []
-
-
-class AbemaTVIE(AbemaTVBaseIE):
-    _VALID_URL = r'https?://abema\.tv/(?P<type>now-on-air|video/episode|channels/.+?/slots)/(?P<id>[^?/]+)'
-    _NETRC_MACHINE = 'abematv'
-    _TESTS = [{
-        'url': 'https://abema.tv/video/episode/194-25_s2_p1',
-        'info_dict': {
-            'id': '194-25_s2_p1',
-            'title': '第1話 「チーズケーキ」　「モーニング再び」',
-            'series': '異世界食堂２',
-            'series_number': 2,
-            'episode': '第1話 「チーズケーキ」　「モーニング再び」',
-            'episode_number': 1,
-        },
-        'skip': 'expired',
-    }, {
-        'url': 'https://abema.tv/channels/anime-live2/slots/E8tvAnMJ7a9a5d',
-        'info_dict': {
-            'id': 'E8tvAnMJ7a9a5d',
-            'title': 'ゆるキャン△ SEASON２ 全話一挙【無料ビデオ72時間】',
-            'series': 'ゆるキャン△ SEASON２',
-            'episode': 'ゆるキャン△ SEASON２ 全話一挙【無料ビデオ72時間】',
-            'series_number': 2,
-            'episode_number': 1,
-            'description': 'md5:9c5a3172ae763278f9303922f0ea5b17',
-        },
-        'skip': 'expired',
-    }, {
-        'url': 'https://abema.tv/video/episode/87-877_s1282_p31047',
-        'info_dict': {
-            'id': 'E8tvAnMJ7a9a5d',
-            'title': '第5話『光射す』',
-            'description': 'md5:56d4fc1b4f7769ded5f923c55bb4695d',
-            'thumbnail': r're:https://hayabusa\.io/.+',
-            'series': '相棒',
-            'episode': '第5話『光射す』',
-        },
-        'skip': 'expired',
-    }, {
-        'url': 'https://abema.tv/now-on-air/abema-anime',
-        'info_dict': {
-            'id': 'abema-anime',
-            # this varies
-            # 'title': '女子高生の無駄づかい 全話一挙【無料ビデオ72時間】',
-            'description': 'md5:55f2e61f46a17e9230802d7bcc913d5f',
-            'is_live': True,
-        },
-        'skip': 'Not supported until yt-dlp implements native live downloader OR AbemaTV can start a local HTTP server',
-    }]
     _USERTOKEN = None
     _DEVICE_ID = None
-    _TIMETABLE = None
     _MEDIATOKEN = None
 
     _SECRETKEY = b'v+Gjs=25Aw5erR!J8ZuvRrCx*rGswhB&qdHd_SYerEWdU&a?3DzN9BRbp5KwY4hEmcj5#fykMjJ=AuWz5GSMY-d@H7DMEh3M@9n2G552Us$$k9cD=3TxwWe86!x#Zyhe'
 
-    def _generate_aks(self, deviceid):
+    @staticmethod
+    def _generate_aks(deviceid):
```

Comment:
any reason to make this a static method?

Reply:
because the instance wasn't used